config/locales/sv.yml

```diff
@@ -788,7 +788,8 @@ sv:
       use_this_image_link_html: "<span style='font-weight: bold'>
                                 Använd den här länken på en extern webbplats:
                                 </span>"
-      show_image: Visa bild
+      show_image: Kopiera bildadress
```

Comment:
Don't change this one because it makes the meaning of the key (`show_image `) different from the meaning of the value (`copy the url`).

Instead, add a new entry (line) below (you'll need to ensure that the indentation is correct):

```
copy_image_url: Kopiera bildadress
```
Likewise for the English translation file (`en.yml`), add the same line but with the English words: `copy_image_url: Copy the URL`

_**Then**_ you need to be sure to use the `users.show.copy_image_url` translation key instead of the old one (old = `users.show.show_image`).
(That includes places in cucumber tests.)